### PR TITLE
Issue 1 - Link to contributing guidelines in README

### LINKED ISSUE #1: Add a "How to contribute" section on main README
We want each other to feel comfortable contributing. Please create a friendly, and easy to follow section, on how others can contribute to this project.

### PR DESCRIPTION
**Pull request by crawfoal**
This PR also makes relatively big changes to the contributing guidelines.

Closes #1

### CODE REVIEW

File: .github/CONTRIBUTING.md
Comment (codeByScott) [inline .github/CONTRIBUTING.md]: Need to fix link. Missing the ]

**Review by codeByScott — APPROVED** [review]
This looks great!

File: .github/CONTRIBUTING.md
Comment (crawfoal) [reply .github/CONTRIBUTING.md]: Done 🙂

**Comment by crawfoal** [comment]
@codeByScott What process should we follow here? Once you approve, I go ahead and merge?

I feel like I remember you mentioning that it would be nice to have each change reviewed by at least two people, but in general I'm not sure who else we have that can review PRs. With this specific PR though, I think anyone else could review it.

What are your thoughts?

**Comment by codeByScott** [comment]
@crawfoal with only two of us working on this currently it will be tough to get two reviews. This will be more important when we have a production app. My initial thought is as long as one of us reviews and approves, go ahead and merge the request. There is only a few of us with merge rights at this time.

When the app goes live, I would like to get at least two reviews before merging into master.

Feel free to merge this one as I have already reviewed and have no suggestions for changes.

**Pull request merged**
